packages/zowe-explorer-api/src/tree/IZoweTreeNode.ts

```diff
@@ -120,6 +125,11 @@ export interface IZoweTreeNode {
     setSessionToChoice(sessionObj: imperative.Session): void;
 }
 
+export type DatasetStats = {
+    user: string;
+    m4date: Date;
```

Comment:
`m4date` is the "date string" that is returned as a stat attribute for each data set - see this example response:

https://www.ibm.com/docs/en/zos/2.5.0?topic=interface-list-members-zos-data-set#ListDataSetMembers__title__12

Reply:
FWIW, I'm personally ok with leaving it as m4date.
At times we may choose to go with the IBM z/OSMF naming conventions so that it's easily searchable.

Reply:
I think it's fair to use `modifiedDate` instead of `m4date`, I'll just add a comment above it to give context on what it's called in the z/OSMF response 🙂